.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+	"*.js": ["eslint", "prettier --write", "git add"]
```

Comment:
Could we make these JSON files use the same indentation and have a line at the end? Sublime Text is letting you down.